src/SFA.DAS.RoatpOversight.Web/Views/Oversight/Applications.cshtml

```diff
@@ -147,8 +158,104 @@
+                                            <td data-label="Application submitted date" class="govuk-table__cell">@outcome.ApplicationSubmittedDate.ToSfaShortDateString()</td>
+                                            <td data-label="Application determined date" class="govuk-table__cell">@outcome.ApplicationDeterminedDate.ToSfaShortDateString()</td>
+                                            <td data-label="Appeal submitted date" class="govuk-table__cell">@outcome.AppealSubmittedDate.ToSfaShortDateString()</td>
+                                            <td data-label="Overall outcome" class="govuk-table__cell">
+                                                <span class="das-no-wrap @outcome.OversightStatus.GetCssClass()">@outcome.OversightStatus.GetLabel()</span>
+                                            </td>
+                                        </tr>
+                                    }
+                                </tbody>
+                            </table>
+                        }
+                        else
+                        {
+                            <p class="govuk-body-l"> No applications to assess.</p>
+                        }
+                    </div>
+                    <div class="govuk-tabs__panel @(Model.SelectedTab != null && Model.SelectedTab != "appealsoutcome" ? "govuk-tabs__panel--hidden" : "")" id="appealoutcome">
+
+                        @if (Model.AppealsOutcomeCount > 0)
+                        {
+                            <table class="govuk-table das-table--condensed das-table--responsive-wide">
+                                <caption class="govuk-table__caption govuk-heading-l">Appeal outcomes</caption>
+                                <thead class="govuk-table__head">
+                                    <tr class="govuk-table__row">
+                                        <th scope="col" class="govuk-table__header">Organisation name</th>
+                                        <th scope="col" class="govuk-table__header">UKPRN</th>
+                                        <th scope="col" class="govuk-table__header">Provider route</th>
+                                        <th scope="col" class="govuk-table__header">Application reference number</th>
+                                        <th scope="col" class="govuk-table__header">Appeal submitted date</th>
+                                        <th class="govuk-table__header" scope="col">
+                                            <sfa-roatp-sortable-column selected-tab="appealsoutcome" column-name="AppealDeterminedDate" column-label="Appeal determined date" default="true" default-order="Descending" />
+                                        </th>
+                                        <th scope="col" class="govuk-table__header">Overall outcome</th>
```

Comment:
Looks like it is missing a matching <td>

Reply:
sorry what it means

Reply:
verified looks alright, let me know if you still feels missing